src/styledmarkup.jl

```diff
@@ -802,20 +803,22 @@ function read_face_or_keyval!(state::State, i::Int, char::Char, newstyles)
         push!(newstyles,
                 (i, i + state.offset + ncodeunits('{'),
                 if key isa String && !(value isa Symbol || value isa Expr)
-                    Pair{Symbol, Any}(Symbol(key), value)
+                    # TODO
```

Comment:
I could use some help filling this logic in here

if `key` might be `:face` and we want that to display properly, we need to force resolution here too